russstation/code/modules/cargo/packs.dm

```diff
@@ -50,3 +50,4 @@
     				/obj/item/toy/gun/bigiron)
 	crate_name = "wild west crate"
 	crate_type = /obj/structure/closet/crate/wooden
+
```

Comment:
random empty line on an otherwise unedited file

Reply:
Thats where the manual insert of the miasma tank was when it was first pred, then i removed it